To: vim_dev@googlegroups.com
Subject: Patch 7.3.1
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.101
Problem:    ino_t defined with wrong size.
Solution:   Move including auto/config.h before other includes. (Marius
            Geminas)
Files:      src/if_ruby.c, src/if_lua.c
	        

*** ../vim-7.3.100/src/if_ruby.c	2010-12-24 13:39:29.000000000 +0100
--- src/if_ruby.c	2011-01-09 14:43:14.000000000 +0100
***************
*** 11,23 ****
   * See README.txt for an overview of the Vim source code.
   */
  
- #include <stdio.h>
- #include <string.h>
- 
  #ifdef HAVE_CONFIG_H
  # include "auto/config.h"
  #endif
  
  #ifdef _WIN32
  # if !defined(DYNAMIC_RUBY_VER) || (DYNAMIC_RUBY_VER < 18)
  #   define NT
--- 11,23 ----
   * See README.txt for an overview of the Vim source code.
   */
  
  #ifdef HAVE_CONFIG_H
  # include "auto/config.h"
  #endif
  
+ #include <stdio.h>
+ #include <string.h>
+ 
  #ifdef _WIN32
  # if !defined(DYNAMIC_RUBY_VER) || (DYNAMIC_RUBY_VER < 18)
  #   define NT
*** ../vim-7.3.100/src/if_lua.c	2010-10-23 14:02:48.000000000 +0200
--- src/if_lua.c	2011-01-09 14:46:46.000000000 +0100
***************
*** 9,20 ****
   * See README.txt for an overview of the Vim source code.
   */
  
! #include <stdio.h>
! #include <string.h>
  #include <lua.h>
  #include <lualib.h>
  #include <lauxlib.h>
- #include "vim.h"
  
  /* Only do the following when the feature is enabled.  Needed for "make
   * depend". */
--- 9,19 ----
   * See README.txt for an overview of the Vim source code.
   */
  
! #include "vim.h"
! 
  #include <lua.h>
  #include <lualib.h>
  #include <lauxlib.h>
  
  /* Only do the following when the feature is enabled.  Needed for "make
   * depend". */
*** ../vim-7.3.100/src/version.c	2011-01-17 19:50:01.000000000 +0100
--- src/version.c	2011-01-17 19:51:40.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     101,
  /**/

-- 
In a world without walls and borders, who needs windows and gates?

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
